src/ergw_aaa_radius.erl

```diff
@@ -46,7 +46,7 @@
     {peer_restart, 7},
     {'ASR', 6},
     {error, 9},
-    {req_timeout, 4},
+    {timeout, 4},
```

Comment:
I suppose you need also update the  README.

Reply:
You also need to change ergw_aaa_diameter module I guess.

Reply:
I'll update the docs. Re: the other modules, I'm not sure really, but I guess it should be consistent :-/ Thanks!!!